plugin/src/main/java/io/fabric8/maven/plugin/mojo/build/ResourceMojo.java

```diff
@@ -234,13 +228,8 @@
     public ResourceMojo() {
     }
 
-    @Override
-    public void execute() throws MojoExecutionException, MojoFailureException {
-        clusterAccess = new ClusterAccess(namespace);
```

Comment:
I knew there was a reason, but as it is now it indeed doesn't make sense to have this here. I blame amnesia for this ;-)

lgtm [merge]